lb_content_resolver/content_resolver.py

```diff
@@ -190,12 +197,29 @@ def add_file_to_index(self, relative_path, format, mtime):
             mdata["mtime"] = mtime
             mdata["file_path"] = file_path
 
-            # now add the record
-            self.index_metadata(mdata)
+            mdata["artist_mbid"] = self.convert_to_uuid(mdata["artist_mbid"])
```

Comment:
I moved this under the `is not None` check to fix a failure I got. Then a small tidy up to remove the repeated code into a method.